src/test/java/school/hei/haapi/integration/TeacherIT.java

```diff
@@ -83,6 +92,32 @@ public void setUp() {
     setUpEventBridge(eventBridgeClientMock);
   }
 
+  @Test
+  void teacher_update_own_profile_picture() throws IOException, InterruptedException {
+    String TEACHER_ONE_PICTURE_RAW = "/teachers/" + TEACHER1_ID + "/picture/raw";
+    HttpClient httpClient = HttpClient.newBuilder().build();
+    String basePath = "http://localhost:" + TeacherIT.ContextInitializer.SERVER_PORT;
+
+    HttpRequest.BodyPublisher body =
+        HttpRequest.BodyPublishers.ofByteArray(getMockedFileAsByte("img", ".png"));
+    HttpResponse<String> response =
+        httpClient.send(
+            HttpRequest.newBuilder()
+                .uri(URI.create(basePath + TEACHER_ONE_PICTURE_RAW))
+                .POST(body)
+                .setHeader("Content-Type", "image/png")
+                .header("Authorization", "Bearer " + TEACHER1_TOKEN)
+                .build(),
+            HttpResponse.BodyHandlers.ofString());
+
+    ObjectMapper mapper = new ObjectMapper();
+    mapper.registerModule(new JSR310Module());
+    mapper.configure(DeserializationFeature.READ_DATE_TIMESTAMPS_AS_NANOSECONDS, false);
```

Comment:
@Autowire an ObjectMapper because EndpointConf already gives you an ObjectMapper bean